create_tables.sql

```diff
@@ -0,0 +1,56 @@
+CREATE EXTENSION IF NOT EXISTS "pg-crypto";
+
+CREATE TYPE STATUS_COLOR AS ENUM ('GREEN', 'ORANGE', 'RED', 'GREY');
+CREATE TYPE DEPLOYMENT_STATUS AS ENUM ('ready-to-deploy', 'not-ready-to-deploy');
```

Comment:
I know I picked these and I'm being annoying, but I'm realizing we probably want
`ready` and `paused` instead -- it's not a case of readiness, but user intention. (Maybe we can brainstorm names together?)